server/database/migrations/16.add-event-confirmation-email-occurrence.js

```diff
@@ -0,0 +1,9 @@
+exports.up = (knex, Promise) =>
+  knex.schema.table('event', (table) => {
+    table.integer('times_confirmation_email_sent').defaultTo(0).comment('Number of times a confirmation email has been sent; Used to shift content of email');
```

Comment:
What does `Used to shift content of email` mean in this context?

Reply:
On the sendgrid side, we're customizing content of email depending on variables.
Each email uses the same template, but makes use of branches to customize the language/icons.
If you have a better rewording in english, I'm a taker :)

Reply:
I am sure there is some, but I cannot grasp it ;)